rfcs/stinkball-database.md

```diff
@@ -24,55 +24,85 @@ Database will store user information, NFL players, fantasy teams, and Stinkball
 {
     "id": "string",
     "player_name": "string",
-    "nfl_team": "string",
+    "nfl_team": "nfl team model",
     "position": "string",
-    "weekly_points": "array of numbers",
 }
 ```
 
 - `id: string` - unique id for player
 - `player_name: string` - name of player
-- `nfl_team: string` - team player plays for
-- `position: string` - position player plays
-- `weekly_points: []` - array of points player has scored each week
+- `nfl_team: nfl team model` - foreign key to nfl team id
+- `position: string` - 
 
 
+**Weekly Points Model**
+```
+{
+    "id": "string",
+    "player_id": "string",
+    "week": "number",
+    "points": "number"
+}
+```
+
+- `id: string` - unique id for weekly points scored
+- `player_id: string` - foreign key to player id
+- `week: number` - week of nfl season
+- `points: number` - number of points scored by player for specified week
+
+
+**NFL Team Model**
+```
+{
+    "id": "string",
+    "team_name_short": "string", - 
+    "team_name_long": "string",
+    "team_city_long": "string",
+    "team_city_short": "string",
+    "team_logo": "string",
+    "team_players": "array of strings",
+}
+```
+- `id: string` - unique id for nfl team
+- `team_name_short: string` - abbreviated nfl team name
+- `team_name_long: string` - full nfl team name
+- `team_city_short: string` - abbreviated nfl team city
+- `team_city_long: string` - full nfl team city
+- `team_logo: string` - url path to stored nfl logo image
+- `team_players: []` - array of player models on nfl team
```

Comment:
Do this the other way around - where the teamId exists on the player model. This is a more appropriate and scalable appraoch